<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Scr_Map</class>
 <widget class="QDialog" name="Scr_Map">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Safejumper</string>
  </property>
  <widget class="QFrame" name="frameLogo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>22</y>
     <width>375</width>
     <height>89</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
  background-color: #ffffff;
  border: 0;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGraphicsView" name="gvLogoRound">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>16</y>
      <width>57</width>
      <height>57</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QGraphicsView
{
  border-image: url(:/imgs/logo2.png);
}</string>
    </property>
    <property name="interactive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="b_LogoBack">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>16</y>
      <width>71</width>
      <height>57</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
  border:0px;
  color: #ff0000;
background: transparent;
border-image: url(:/imgs/logo3-norm.png);
}
QPushButton:hover
{
border-image: url(:/imgs/logo3.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="L_1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>13</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	border:0px;
	color: #ffffff;
border-image: url(:/imgs/l-1.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="L_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	border:0px;
	color: #ffffff;
border-image: url(:/imgs/l-2-inactive.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="dd_Protocol">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>13</y>
      <width>210</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox
{
color: #444444;

border-image: url(:/imgs/dd-protocol.png);
}
QComboBox QAbstractItemView 
{
subcontrol-origin: margin;
subcontrol-position: top left;
}
QComboBox::drop-down 
{
width: 25px;
border-top-right-radius: 3px;
border-bottom-right-radius: 3px;
}
QComboBox::down-arrow 
{
image: none;
width: 25px;
height: 25px;
}
QComboBox::down-arrow:hover
{
image: none;
width: 25px;
height: 25px;
}

QComboBox 
{
	padding-left: 10px;
	padding-right: 0px; /* space for the arrow */
}

QComboBox::drop-down 
{
	subcontrol-origin: top;
	subcontrol-position: top right;
	width: 0px;
	border-style: none;
	border-image: none;
}</string>
    </property>
    <property name="currentText">
     <string>-- Select protocol --</string>
    </property>
    <property name="maxVisibleItems">
     <number>13</number>
    </property>
    <item>
     <property name="text">
      <string>-- Select protocol --</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="dd_Location">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>50</y>
      <width>210</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox
{
color: #444444;

border-image: url(:/imgs/dd-protocol.png);
}
QComboBox QAbstractItemView 
{
subcontrol-origin: margin;
subcontrol-position: top left;
}
QComboBox::drop-down 
{
width: 25px;
border-top-right-radius: 3px;
border-bottom-right-radius: 3px;
}
QComboBox::down-arrow 
{
image: none;
width: 25px;
height: 25px;
}
QComboBox::down-arrow:hover
{
image: none;
width: 25px;
height: 25px;
}

QComboBox {
	padding-left: 10px;
	padding-right: 0px; /* space for the arrow */
}

QComboBox::drop-down {
	subcontrol-origin: top;
	subcontrol-position: top right;
	width: 0px;
	border-style: none;
	border-image: none;
}
</string>
    </property>
    <item>
     <property name="text">
      <string>-- Select location --</string>
     </property>
    </item>
   </widget>
   <widget class="QListView" name="lv_Location">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>350</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QListView
{
background: solid #f7f7f7;
}
QListView::item
{
/*color: #202020;*/
color: #f7f7f7;	/*invisible*/
height: 24;
border-image: url(:/imgs/dd-row.png);
background: solid #f7f7f7;
}
QListView::item:checked
{
border-image: url(:/imgs/e-error-241.png);
}
QListView::item:active
{
border-image: url(:/imgs/e-error-241.png);
}
QListView::item:selected
{
border-image: url(:/imgs/dd-selectionrow.png);
}

</string>
    </property>
   </widget>
   <widget class="QListView" name="lv_Protocol">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>210</width>
      <height>230</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QListView
{
background: solid #f7f7f7;
}
QListView::item 
{
color: #202020;
height: 24;
border-image: url(:/imgs/dd-row.png);
background: solid #f7f7f7;
}
QListView::item:selected
{
color: #FFFFFF;
border-image: url(:/imgs/dd-selectionrow.png);
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_x">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_y">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frameBottom">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>111</y>
     <width>375</width>
     <height>251</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>247</red>
         <green>247</green>
         <blue>247</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>247</red>
         <green>247</green>
         <blue>247</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>247</red>
         <green>247</green>
         <blue>247</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>247</red>
         <green>247</green>
         <blue>247</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>247</red>
         <green>247</green>
         <blue>247</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>247</red>
         <green>247</green>
         <blue>247</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>247</red>
         <green>247</green>
         <blue>247</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>247</red>
         <green>247</green>
         <blue>247</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>247</red>
         <green>247</green>
         <blue>247</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
  background-color: #f7f7f7;
  border: 0;
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="line_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>375</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
  border:0px;
  border-image: url(:/imgs/h-line.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="line_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>185</y>
      <width>375</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	border:0px;
	border-image: url(:/imgs/h-line.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="b_Settings">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>200</y>
      <width>110</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border:0px;
	color: #ffffff;
	border-image: url(:/imgs/b-settings-norm.png);
}
QPushButton:hover:!pressed
{
	border-image: url(:/imgs/b-settings-hover.png);
}
QPushButton:pressed
{
	border-image: url(:/imgs/b-settings-pressed.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="b_Connect">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>110</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border:0px;
	color: #ffffff;
	border-image: url(:/imgs/b-connect-norm.png);
}
QPushButton:hover:!pressed
{
	border-image: url(:/imgs/b-connect-hover.png);
}
QPushButton:pressed
{
	border-image: url(:/imgs/b-connect-pressed.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="L_Map">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>7</y>
      <width>375</width>
      <height>177</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	border:0px;
	color: #ffffff;
border-image: url(:/imgs/map.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="L_Mark">
    <property name="geometry">
     <rect>
      <x>139</x>
      <y>30</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	border:0px transparent;
	color: #ffffff;
border-image: url(:/imgs/map-border.png);
background: transparent;
padding-bottom: 7px;
}</string>
    </property>
    <property name="text">
     <string>Germany</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="b_Tmp">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border:0px;
	border-image: url(:/imgs/cb-empty.png);
}</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frameHead">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>375</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
	background-color: #ffffff;
	border: 0;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="line_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>21</y>
      <width>375</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	border:0px;
	border-image: url(:/imgs/h-line-mb.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="L_Caption">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>3</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Safejumper</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="b_Head">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>375</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border:0px;
	border-image: url(:/imgs/cb-empty.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="b_Minimize">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>6</y>
      <width>10</width>
      <height>10</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border:0px;
	border-image: url(:/imgs/b-min.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="b_Cross">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>6</y>
      <width>10</width>
      <height>10</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border:0px;
	border-image: url(:/imgs/b-cross.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>b_LogoBack</sender>
   <signal>clicked()</signal>
   <receiver>Scr_Map</receiver>
   <slot>ToScr_Connect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b_Settings</sender>
   <signal>clicked()</signal>
   <receiver>Scr_Map</receiver>
   <slot>ToScr_Settings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dd_Protocol</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Scr_Map</receiver>
   <slot>Changed_dd_Protocol(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dd_Location</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Scr_Map</receiver>
   <slot>Changed_dd_Sever(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b_Connect</sender>
   <signal>clicked()</signal>
   <receiver>Scr_Map</receiver>
   <slot>Clicked_b_Connect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b_Tmp</sender>
   <signal>pressed()</signal>
   <receiver>Scr_Map</receiver>
   <slot>Clicked_b_Tmp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>e_x</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Scr_Map</receiver>
   <slot>Changed_xy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>e_y</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Scr_Map</receiver>
   <slot>Changed_xy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b_Head</sender>
   <signal>pressed()</signal>
   <receiver>Scr_Map</receiver>
   <slot>Pressed_Head()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b_Minimize</sender>
   <signal>clicked()</signal>
   <receiver>Scr_Map</receiver>
   <slot>Clicked_Min()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b_Cross</sender>
   <signal>clicked()</signal>
   <receiver>Scr_Map</receiver>
   <slot>Clicked_Cross()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
